### ISSUE TITLE
Game ends right after picking Hero/Character

### ISSUE THREAD

**Issue by CuddlyE** [Alzarath/Crash-RPG#9]
So, I'm assuming you've long abandoned this project, consider it was made so long ago. However, I have fond memories of playing the origin CrashRPG with my brother in Starcraft. Ever since SCII came out, I went in to check the 'Arcade' for a remake or port of the map into SCII, so I could invite my brother and we could relive the experience. I never found one. I kept checking in regularly years later even when I stopped playing the game. Just to see if someone had made it and uploaded it. Finally today I decided to look more into it, through googling, I found this project, exactly what I was looking for. However, sadly, after downloading the map, and opening it in Map editor, and testing it. I found it to be broken (probably something that broke over years of patches). 

Now, I don't have map making knowledge enough to fix it. However, since choosing a character kills your initial person, and the game ends after a message of 'everybody died'. I'm assuming it's a simple issue of that brief moment between your 'picker' character dying, and your chosen character spawning/going under your control (whatever happens under the hood), the game considers all characters dead, and considers it a game over before it registers that you do in fact have a unit alive. Now, this might not happen with more players, if they pick one at a time, and wait for the other character to spawn before picking. However, without knowing for sure, and without uploading it to test it in game with other players (which might break it further), its hard to know.

I guess this is just a random last ditch effort attempt at realising the dream, by hoping you see this, and feel inclined to check up on the map again, and see if there's any easy fix to make it work with whatever new update broke it. It's a lot to ask, you probably haven't touched the editor in years, or possibly even the game. However, I'd probably regret not trying at least.
![temp](https://user-images.githubusercontent.com/77785537/116707789-5a1b6000-a9cf-11eb-9c39-f0edf172c7d1.png)

**Comment by Alzarath** [comment]
It's true I haven't worked on the map for a long time. But I do not consider it abandoned. Another collaborative map project of mine has taken up most of my time. I really do want to return to the map one day and continue updating it. There's been some half-implemented features for a long time now that would probably just as breaking, otherwise I would push what I have. Maybe I can force myself to at least get it stable and update it with that for the time being. I'll update if/when I do.

In the meantime, the Github repo of the map is out of date. There's a version of the map published to Battle.net in US that's more up-to-date. It should be functional, but I haven't played it in a while either. It's unlocked, so you should be able to download it through the editor. I'd prefer to publish it to other regions myself, but I won't hold you back if you decide to, I just ask that you keep the original name available for me.

**Comment by Alzarath** [comment]
The repo has been updated with the test version. It is not considered fully playable, so use at your own risk, but you should be able to play it fully, with the limitation that Aero is really the only Hero currently able to be played properly.

Admittedly I wanted to make a branch with this, but I made a mistake and am too lazy to be bothered. If you want to play it properly, the live version in NA is still the recommended method. You can find more information on it [here](https://sc2arcade.com/map/1/289773/). More work will be put toward the map in the future.

**Comment by CuddlyE** [comment]
Since I posted this as an 'issue' on the github, I think the best way to
reach back is through email. Thanks for the reply! I'm glad to hear you
haven't abandoned it, I have waited to see a version of it in SC II many
years already, so I don't mind waiting more. It's more important that you
finish it properly in your own time, when your feeling up to working on it,
so I implore you to just keep at it in your own time, instead of pushing
yourself to get out a semi working thing just because I asked!

I at first thought it was a dead project, that didn't work in current
version, and could be fixed with mild adjustments by someone who knows the
map (I.e the one who made it). Now that I know it's still an ongoing
project, I'm fully for simply waiting. As long as I know where to keep up
with it. In worst case, I got the name of you the creator now (of the port
not the original), so in the future, since I know the Github is out of
date, I can look up your name and the map name to check in on progress.

Also, I'm in Europe, and playing on European Servers (obviously), so that
probably explains why I never found it before! I'm not sure how possible it
is to create a lobby with the map from my European account, if its on the
US servers. But I can look into it, and if not, I'll simply wait again, and
check in with the progress :). I wish you luck, and thanks again for not
only responding, but responding so quickly! I honesty did not expect to
hear anything at all, and if so, in some weeks time or something.
